src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/kuntarekry/KuntaRekryTestsIT.java

```diff
@@ -0,0 +1,191 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.kuntarekry;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertEquals;
+
+import java.time.ZoneId;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.otavanopisto.kuntaapi.server.integrations.kuntarekry.KuntaRekryConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.PtvConsts;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+@SuppressWarnings ("squid:S1192")
+public class KuntaRekryTestsIT extends AbstractIntegrationTest {
+  
+  private static final ZoneId TIMEZONE_ID = ZoneId.of("Europe/Helsinki");
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    createPtvSettings();
+    
+    getPtvMocker()
+      .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e")
+      .startMock();
+    
+    getKuntarekryMocker()
+      .mockKuntaRekryFeed("/kuntarekry", "kuntarekry/feed.xml")
+      .startMock();
+    
+    waitApiListCount("/organizations", 1);
+    
+    createKuntaRekrySettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/jobs", getOrganizationId(0)), 3); 
+    
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteKuntaRekrySettings(organizationId);
+  }
+   
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+  
+ private void createKuntaRekrySettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "createKuntaRekrySettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)